packages/components/src/modal/index.tsx

```diff
@@ -177,12 +179,9 @@ function UnforwardedModal(
 		onPointerUp: React.PointerEventHandler< HTMLDivElement >;
 	} = {
 		onPointerDown: ( event ) => {
-			if ( event.isPrimary && event.target === event.currentTarget ) {
+			if ( event.target === event.currentTarget ) {
```

Comment:
Why is the `isPrimary` check being removed, after it was added recently in #52994?

Reply:
Oh I meant to remove that in dbed083 from that PR but goofed and only removed it from the `pointerup`. More context https://github.com/WordPress/gutenberg/pull/52994#issuecomment-1675241542
>I'd been supposing it might be a good to cover a case on multi input devices (think touch screen laptop with mouse) but I think that’s over doing it and I don’t have a device like that for testing anyway.